osu.Game/Screens/Edit/Compose/Components/SelectionRotationHandler.cs

```diff
@@ -27,6 +27,12 @@ public partial class SelectionRotationHandler : Component
         /// </summary>
         public Bindable<bool> CanRotateAroundPlayfieldOrigin { get; private set; } = new BindableBool();
 
+        /// <summary>
+        /// Implementation-defined origin point to rotate around when no explicit origin is provided.
+        /// This field is only assigned during a rotation operation.
+        /// </summary>
+        public Vector2? DefaultOrigin { get; protected set; }
```

Comment:
Why does the rotation handler expose this but the scale handler doesn't? I see no reason to add this member. You could have just assigned to `defaultOrigin` where it used to be and things should work fine no?

Reply:
It's used by the selection box rotation handle to determine the mouse position relative to the selection centre. Recalculating the selection centre in the rotation handle seemed like a messier solution because it doesn't have the selected items and it doesn't know the implementation-specific selection centre.

The selection box scale handle doesn't need this because it uses a custom origin based on which anchor you drag.